### ISSUE TITLE
Can't see the emojis with the "native" style.

### ISSUE THREAD

**Issue by Vrakfall** [morkro/atom-emoji-syntax#32]
Hello,

I'm unable to see the emojis if I let the native style in the settings. Instead, I only see squares. All the other styles work fine.
I'm under Linux Mint 17.3, currently.
![screenshot from 2016-10-19 15 57 44](https://cloud.githubusercontent.com/assets/6759520/19521850/8683b516-9615-11e6-9e5b-fdc36d82128c.png)

**Comment by morkro** [comment]
Nice catch! I don't have Linux, but will try to reproduce and fix the issue 😬

**Comment by Vrakfall** [comment]
A quick vm should do the trick. ;D

**Comment by morkro** [comment]
Just a short update: I did not have found the time to work on that issue, but it is still on my radar!

**Comment by Vrakfall** [comment]
No problemo. Thank you to still think about it. :D

Btw, I think this might be due to the font lacking in the distro. Could it be?

**Comment by Vrakfall** [comment]
Oh and also a quick update: I changed my distro to manjaro-bspwm (which is based on archlinux) and it's still doing it.

I know one of my classmates doesn't have the issue with manjaro-kde.

**Comment by Vrakfall** [comment]
Still doing it. :/ Good news, I think I might have a bit of time to sort it out, in 2 weeks. ;D

**Comment by morkro** [comment]
That would be really awesome! If you need any help on this, just ping me. ✌🏼
Unfortunately, I had not the time yet to properly setup a VM for debugging. I haven't forgotten about this :)

**Comment by dmxt** [comment]
Yeah I have zip zero emoji support on Windows 8 (and for 7 and Linux users as well). All I got is the
